mountpoint-s3-fs/src/metrics.rs

```diff
@@ -310,4 +359,85 @@ mod tests {
             }
         });
     }
+
+    #[test]
+    #[ignore] // Mark as ignored so it doesn't run in normal test runs
```

Comment:
This test appears to just export metrics and then not have any validation steps. Ideally tests should fail if they don't work properly